### PR TITLE
Initialize the 2023 branch

### PR DESCRIPTION
**Pull request by dduportal**
- Bump Docker base images
- Chaotic NPM update run with `ncu -u` (I bet I'll have to remove some of these upgrades, but it's a first blind shot)
- Bump the year and dates (or removes when not clear)

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -8,6 +8,8 @@ RUN apk add --no-cache \
   git \
   tini
 
+RUN npm install --global npm husky
```
Comment (jlevesy) [inline Dockerfile]: (Question, Non Important) Is it necessary to install a git hook library in a docker built image?

**Review by jlevesy — APPROVED** [review]
Sweet, thanks a lot. 

Minor question, but LGTM

File: Dockerfile (hunk 1 above)
Comment (dduportal) [reply Dockerfile]: to be transparent: It was marked as a missing dependency but I did not checked what it is. Shame on me 😅
WiP.

**Comment by dduportal** [comment]
I'm gonna split this PR (bump Node/NPM, and start 2023 content) : consider it a draft.

File: Dockerfile (hunk 1 above)
Comment (dduportal) [reply Dockerfile]: Done in https://github.com/cicd-lectures/slides/pull/80. Husky is a dependency of NPM Check Updater `ncu`.

**Comment by dduportal** [comment]
- Updated this PR for only the content change (2023 and date/times)
- Opened #80 for dependency updates

**Review by jlevesy — APPROVED** [review]

**Pull request merged**